routes/index.js

```diff
@@ -21,10 +22,23 @@ module.exports = function(app) {
         }
       })
         .then(result => {
-          if(result.csvResponse && result.csvResponse == true) {
-            res.setHeader('Content-disposition', 'attachment; filename='+result.fileName);
-            res.set('Content-Type', 'text/csv');
-            res.status(result.status ? result.status : 200).send(result.data);
+          if( result.isResponseAStream == true) {
+              // Check if file specified by the filePath exists 
+            fs.exists(result.fileNameWithPath, function(exists){
+              if (exists) {     
+
+                res.setHeader('Content-disposition', 'attachment; filename='+result.fileNameWithPath.replace("./public/csvFileBackup/",""));
```

Comment:
We should not use the exact string "public/csvFileBackup" here, just extract the last element of the array after split with "/"